thread-manager/src/lib.rs

```diff
@@ -17,7 +17,7 @@ pub use {
     tokio_runtime::{TokioConfig, TokioRuntime},
 };
 
-pub const MAX_THREAD_NAME_CHARS: usize = 12;
+pub const MAX_THREAD_NAME_CHARS: usize = 16;
```

Comment:
Should probably be a separate PR, but since it's a separate commit, I'll allow it 🚓

Reply:
Yeah this ended up being insufficient for the names currently used in agave.